schema_utils.py

```diff
@@ -221,6 +242,17 @@ class _Validators(object):
     """
     @classmethod
     def get(cls, validator_id):
+        """Returns the validator method corresponding to the specified
+        validator_id.
+
+        Args:
+            validator_id: str. The name of the validator method that should be
+                retrieved.
+
+        Returns:
+            function. The validator method corresponding to the specified
+                validator_id.
+        """
```

Comment:
In the functions below it looks like the "Args" and "Returns" fields are missing. Please fix these too; in general, proofread the whole file to make sure that the docstrings are correct.

Reply:
Done